hapic/hapic.py

```diff
@@ -428,7 +423,7 @@ def add_documentation_view(
             description=description,
         )
 
-        def spec_yaml_view():
+        def spec_yaml_view( *args, **kwargs):
```

Comment:
what is the goal of adding  `*args, **kwargs` arguments?

Reply:
It's about compatibility with all framework. This methods will be used by the framework, flask will not pass any argument when pyramid will automatically try to do it. That's why we should allow arguments there.

Reply:
this explanation should be added as a comment

Reply:
@inkhey did you add a comment about the *args, **kwargs ? After that we'll be able to merge